chrome/background.js

```diff
@@ -582,8 +582,7 @@ passwordalert.background.handleKeypress_ = function(tabId, request) {
   }
 
   for (var i = 1; i < passwordalert.background.passwordLengths_.length; i++) {
```

Comment:
In order to make this more timing independent, then we'd also need change the looping. Otherwise we're looking from 1...password_length and that would have similar timing implications. Perhaps something like
for (var i = 1; i < 100; i++) {
  request.password = passwordalert.background.tabState_[tabId]['typedChars'].substr(-1 * i);
  passwordalert.background.checkPassword_(tabId, request, false);
}